desktop/components/article_figure/mixin.jade

```diff
@@ -16,11 +21,11 @@ mixin article-figure(article, width, showDate, showShare)
         = article.get('thumbnail_title')
 
       .article-figure-author-section
-        if article.get('contributing_authors').length === 0 || article.get('channel_id') !== sd.ARTSY_EDITORIAL_CHANNEL || sd.CURRENT_PATH !== '/articles'
-          .article-figure-author(class = article.get('contributing_authors').length > 0 ? 'has-contributing-author' : '')
+        if article.get('contributing_authors') && article.get('contributing_authors').length === 0 || article.get('channel_id') !== sd.ARTSY_EDITORIAL_CHANNEL || sd.CURRENT_PATH !== '/articles'
```

Comment:
Since this conditional is so long, can you add a in-plain-words comment above describing whats going on?